DOCS-2999 Create parallel persistent L2ARC article for SCALE (#1217)
truenas/documentation: content/SCALE/Notices/PersistentL2ARCinSCALE.md

---

> title: "Persistent L2ARC in TrueNAS 12.0"

Reviewer: Rename for SCALE (12.0 doesn't apply)
```suggestion
title: "Persistent L2ARC in TrueNAS SCALE"
```

weight: 20
---

> Persistent L2ARC, or L2ARC rebuild, is a ZFS feature that was added to TrueNAS 12.0.

Reviewer: ```suggestion
Persistent L2ARC, or L2ARC rebuild, is a ZFS feature that was added to TrueNAS 12.0 and is also present in TrueNAS SCALE.
```

By default, the L2ARC cache empties when the system reboots.
When persistent L2ARC is enabled, a sysctl repopulates the cache device mapping during the reboot process.
This is intended to preserve L2ARC performance even after a system reboot.

However, persistent L2ARC for very large data pools can drastically slow the reboot process, degrading middleware and web interface performance.

> Because of this, persistent L2ARC is disabled by default in TrueNAS 12.0 and must be manually activated.

Reviewer: ```suggestion
Because of this, persistent L2ARC is disabled by default in TrueNAS SCALE and must be manually activated.
```

## Activating Persistent L2ARC

> Go to **System > Tunables > ADD**.

Reviewer: Adjusting for SCALE UI placement (reviewed SCALE 22.02-RC.2 system)
```suggestion
Go to **System Settings > Advanced**, find the **Sysctl** card and click **Add**.
```

> For the *Variable*, enter `vfs.zfs.l2arc.rebuild_enabled`. Set the *Value* to *1* and the *Type* to *sysctl*.

Reviewer: Type field is not present in SCALE.
```suggestion
For the *Variable*, enter `vfs.zfs.l2arc.rebuild_enabled`. Set the *Value* to *1*.
```

It is often helpful to note in the *Description* that this is the Persistent L2ARC activation.

> Make sure *Enabled* is set and click *SUBMIT*.

Reviewer: ```suggestion
Make sure *Enabled* is set and click *Save*.
```

> ![PersistentL2ARCTunable](/images/CORE/12.0/SystemTunablesL2ARCRebuild.png "Persistent L2ARC Activation")

Reviewer: Rewrite for SCALE image use.
```suggestion
![PersistentL2ARCSCALE](/images/SCALE/PersistentL2ARCSCALE.png "Persistent L2ARC Activation")
```
